src/PKSim.Core/Model/IndividualProtein.cs

```diff
@@ -16,8 +17,33 @@ public enum IntracellularVascularEndoLocation
       Interstitial
    }
 
+   [Flags]
+   public enum Localization
```

Comment:
Localization is defined as a flag so that I can do some nice boolean magic with it